cylc/sphinx_ext/rtd_theme_addons/_static/css/addons.css

```diff
@@ -119,3 +119,10 @@ span.versionmodified{
     overflow-x: hidden!important;
     overflow-y: hidden!important;
 }
+
+
+/* (8)
+ * Fix zero height auto size of svg */
+.logo {
+    height: 110px!important;
```

Comment:
Where possible try to use `em`s or `%`s so that changes in the theme don't change the style.